app/src/main/java/io/gnosis/safe/ui/settings/app/AppSettingsViewModel.kt

```diff
@@ -23,6 +25,7 @@ class AppSettingsViewModel @Inject constructor(
 
     fun removeSigningOwner() {
         ownerCredentialsRepository.removeCredentials()
+        tracker.setNumKeysImported(0)
```

Comment:
Should I add a test that verifies that `tracker.setNumKeysImported(0)` was called?

Reply:
If there is already a test setup just add a `verify` call otherwise, I would say that you don't need to bother.